src/ZODB/Connection.py

```diff
@@ -1328,7 +1328,7 @@ def note(self, text): # for tests
     @property
     def _extension(self):
         warnings.warn("_extension is deprecated, use extension",
-                      DeprecationWarning)
+                      DeprecationWarning, stacklevel=2)
```

Comment:
This change looks unrelated to the PR.

Reply:
In order to solve the DeprecationWarning, I needed to know where it was coming from. Without specifying the stacklevel, the warning appears to be coming from this code, not from the place that's actually triggering the warning.

Reply:
OIC